frontend/src/routes/routes.js

```diff
@@ -10,6 +10,7 @@ const routes = [
   },
   {
     path: '/profile',
+    name: 'Profile',
```

Comment:
@Mogge do we use this? @ogerly How is this related to the PR?

Reply:
No, so far we do not have named views.